frontend/src/store/modules/place/mutations.js

```diff
@@ -18,4 +18,16 @@ export default {
     SET_CURRENT_PLACE_RATING_COUNT: (state, ratingCount) => {
         state.currentPlace.ratingCount = ratingCount;
     },
+    SET_CURRENT_PLACE_REVIEW_LIKE_STATE: (state, { reviewId, likeState }) => {
+        const review = state.currentPlace.reviews.find(function (review) { return review.id === parseInt(reviewId); });
```

Comment:
reviews should have 
{
  byId: {},
  allIds: []
}
structure,
So search by id would be much easier than `reviews.find(function (review) { return review.id === parseInt(reviewId); });`